membership-attribute-service/app/repositories/MembershipAttributesSerializer.scala

```diff
@@ -24,13 +26,16 @@ case class MembershipAttributesSerializer(tableName: String)
     Map(
       AttributeNames.userId -> membershipAttributes.UserId,
       AttributeNames.membershipNumber -> membershipAttributes.MembershipNumber.getOrElse(""),
-      AttributeNames.tier -> membershipAttributes.Tier
+      AttributeNames.tier -> membershipAttributes.Tier,
+      AttributeNames.startDate -> membershipAttributes.StartDate.map(_.toString).getOrElse("")
     ).filter(_._2.nonEmpty).map(mkAttribute[String])
 
-  override def fromAttributeMap(item: collection.mutable.Map[String, AttributeValue]) =
+  override def fromAttributeMap(item: collection.mutable.Map[String, AttributeValue]) = {
     Attributes(
       UserId = item(AttributeNames.userId),
       MembershipNumber = item.get(AttributeNames.membershipNumber).map(_.getS),
-      Tier = item(AttributeNames.tier)
+      Tier = item(AttributeNames.tier),
+      StartDate = item.get(AttributeNames.startDate).map(LocalDate.parse(_))
```

Comment:
@svillafe this is the `.get` call needed to stop the exception, but sounds like Jacob's going to delete it anyway :1st_place_medal: